### PR TITLE
Fix Tutorials

### PR DESCRIPTION
**Pull request by MetRonnie**
- Ensured that every tutorial suite using the `get-observations` script has an `etc/` symlink pointing to `cylc-forecasting-suite/etc/`, so that they can use the `met-office-sites.dat` file within
- Replaced Belmullet with Aldergrove

### CODE REVIEW

**Review by wxtim — APPROVED** [review]

File: sphinx/tutorial/cylc/runtime/configuration-consolidation/index.rst [hunk 1]
```diff
@@ -68,7 +68,7 @@ Consolidating Configuration
           [[dependencies]]
               [[[T00/PT3H]]]
                   graph = """
-                      get_observations_belmullet => consolidate_observations
+                      get_observations_aldergrove => consolidate_observations
                       get_observations_camborne => consolidate_observations
                       get_observations_heathrow => consolidate_observations
                       get_observations_shetland => consolidate_observations
```
Comment (oliver-sanders) [inline sphinx/tutorial/cylc/runtime/configuration-consolidation/index.rst:74]: FYI: The original list of sites definitely weren't selected solely on the basis of line length.

File: sphinx/tutorial/cylc/runtime/configuration-consolidation/parameters.rst [hunk 2]
```diff
@@ -248,7 +248,7 @@ Parameters can be either words or integers:
 
          The relevant IDs are:
 
-         * Belmullet - ``3976``
+         * aldergrove - ``3917``
```
Comment (oliver-sanders) [inline sphinx/tutorial/cylc/runtime/configuration-consolidation/parameters.rst:251]:
```suggestion
         * Aldergrove - ``3917``
```

File: sphinx/tutorial/cylc/runtime/runtime-configuration.rst [hunk 3]
```diff
@@ -334,7 +334,7 @@ Start, Stop, Restart
 
       * Camborne - ``3808``
       * Shetland - ``3005``
-      * Belmullet - ``3976``
+      * aldergrove - ``3917``
```
Comment (oliver-sanders) [inline sphinx/tutorial/cylc/runtime/runtime-configuration.rst:337]:
```suggestion
      * Aldergrove - ``3917``
```

**Review by oliver-sanders — APPROVED** [review]
confirmed `t/docs/02-tutorial-suites.t` passing locally

File: sphinx/tutorial/cylc/runtime/configuration-consolidation/index.rst (hunk 1 above)
Comment (wxtim) [reply sphinx/tutorial/cylc/runtime/configuration-consolidation/index.rst:74]: You're lucky I didn't try to sneak 03257 (RAF Leeming) in there.

**Pull request merged**